### ISSUE TITLE
Segfault on bcftools expressions using "=>" instead of ">="

### ISSUE THREAD

**Issue by tseemann** [samtools/bcftools#770]
My fault ends up being the parser's seg-fault :-)

```
# typo in AF=>0
% bcftools view --include 'GT="1/1" && DP>=0 && AF=>0' snps.vcf > /dev/null
(segmentation fault)

# correct syntax
% bcftools view --include 'GT="1/1" && DP>=0 && AF>=0' snps.vcf > /dev/null
```

```
bcftools --version
bcftools 1.8
Using htslib 1.8
```

**Comment by tseemann** [comment]
@pd3 may i ask what the outcome of this was?

**Comment by pd3** [comment]
Ah, sorry for not commenting here. This should be fixed by 5bab0c5. Thank you for reporting the bug!
